codegen/runner/runner.go

```diff
@@ -70,7 +70,7 @@ func main() {
 		config.MustGetString("clientConfig"),
 		config.MustGetString("endpointConfig"),
 		config.MustGetString("middlewareConfig"),
-
+		config.MustGetString("copyrightHeader"),
```

Comment:
I wanted this to be definable by the user so that if someone uses the open sourced zanzibar to generate a gateway, then the files build based on their specifications would have their copyright attached instead of the zanzibar/uber copyright header (just for the generated files, not for the generators or runtime).